Extending/Packages/Packages-on-Umbraco-Cloud/index.md

```diff
@@ -0,0 +1,287 @@
+---
+versionFrom: 8.0.0
+---
+
+# Packages on Umbraco Cloud
+
+If you want to use or develop packages for Umbraco Cloud there are a few things to consider and be aware of. One such thing is that custom property editors may need a **ValueConnector** to transform their content between environments.
+
+# ValueConnectors
+
+A ValueConnector is an extension to Deploy that allows you to transform data when you deploy content of any kind between environments. It is mostly used to transfer ID based content between environments.
```

Comment:
You may want to add in that `ValueConnectors` are also responsible for resolving dependencies. Resolving dependencies is a just as important role of `ValueConnectors` as transforming the values is.